--- needrestart-2.0/ex/needrestart.conf	2015-04-14 22:56:40.000000000 +0300
+++ needrestart-2.0/ex/needrestart.conf	2014-12-03 00:00:40.000000000 +0200
@@ -29,10 +29,10 @@
 
 # Disable sending notifications to user sessions running obsolete binaries
 # using scripts from $nrconf{notify_d}.
-#$nrconf{sendnotify} = 0;
+$nrconf{sendnotify} = 0;
 
 # Restart services (l)ist only, (i)nteractive or (a)utomatically.
-#$nrconf{restart} = 'i';
+$nrconf{restart} = 'l';
 
 # Use prefered UI package.
 #$nrconf{ui} = 'NeedRestart::UI::stdio';
